gamebridge_reshade/src/igraphicsapi.h

```diff
@@ -6,24 +6,19 @@
  */
 
 #pragma once
+
 #include "pch.h"
 
 // List of color formats that require color linearization in order restore the correct gamma.
 const std::list<reshade::api::format> srgb_color_formats {
     reshade::api::format::r8g8b8a8_unorm_srgb,
 };
 
-const uint32_t DEFAULT_WEAVER_LATENCY = 40000;
+const uint32_t g_default_weaver_latency = 40000;
```

Comment:
This is a constant, not a global

Reply:
Fixed in commit: c7db2deb5f57c54e8ef975fd5065ca13bd2bd4fb